sportsreference/ncaaf/teams.py

```diff
@@ -89,7 +111,7 @@ def _parse_team_data(self, team_data):
         """
         for field in self.__dict__:
             if field == '_year' or \
-               field == '_team_conference':
+                    field == '_team_conference':
```

Comment:
Just curious - was this padding intentional? If so, and you prefer this way, works for me! If not, would you be able to revert back to the original spacing?

Reply:
Sorry about this one - I think I had PyCharm autoreformat the file or something. I'll revert this change.